core/src/main/scala/zio/logging/Logging.scala

```diff
@@ -27,32 +20,29 @@ object Logging {
   def error(cause: Cause[Any]): ZIO[Logging, Nothing, Unit] =
     log(LogLevel.Error)(cause.prettyPrint)
 
-  val ignore: UIO[Logging] =
+  val ignore: ZLayer[Any, Nothing, Logging] =
```

Comment:
Very minor but could be slightly more idiomatically `ZLayer.NoDeps[Nothing, Logging]`.

Reply:
thanks I was looking for that :)